Sources/MessagingPushAPN/MessagingPushAPN+PushConfigs.swift

```diff
@@ -0,0 +1,35 @@
+import Foundation
+import UIKit
+
+extension MessagingPushAPN {
+    func setupAutoFetchDeviceToken() {
+        // Register for push notifications
+        UIApplication.shared.registerForRemoteNotifications()
+        // Swizzle method `didRegisterForRemoteNotificationsWithDeviceToken`
+        swizzleDidRegisterForRemoteNotifications()
```

Comment:
```suggestion
        swizzleDidRegisterForRemoteNotifications()
        UIApplication.shared.registerForRemoteNotifications()
```

Shouldn't these 2 function calls be switched in order? If not, I think a comment explaining why this is the order would be helpful. 

I was expecting that the SDK needed to swizzle *first* in order to receive the callback when `registerForRemoteNotifications` gets called?

Reply:
Thanks for the feedback. I do not see any difference in functionality, however I have no issue to swizzle first and then register for remote notifications. Order changed.